### DISCUSSION TITLE
Potential use in PCBox

### DISCUSSION THREAD

**Discussion by bachimusprime** [skjelten/emusc#66] [General]
Hey, I'm the maintainer of PCBox, and I'd like to be able to use libEmuSC in my emulator, but the problem is that it's LGPLv3+ instead of a license compatible with GPLv2, which is what PCBox is under. Do you think there's anything you guys could do about that? Thanks.

**Comment by skjelten** [comment]
This is a questions that pops up from time to time. I should probably just re-license libEmuSC under the LGPLv2.1+ license to reach a wider audience. But for to do that I need approval from @mmontag as he has done a few significant contributions.

That said, I see your project is licensed under the GPLv2+ license ("PCBox is a low-level PC emulator based on 86Box. It is licensed under GPLv2 or later."), so it should be able to use libEmuSC today, even though it of course will make PCBox GPLv3 when linked to libEmuSC - which can cause problems if you also link with e.g. GPL2 libraries.

I am very happy that people want to use libEmuSC, but I have to warn that the API is neither documented nor well thought through at this time. I have focused 100% of my time on making a working emulator that can reproduce best possible audio so far, and have not spent any time on the API yet. It should not be too hard to get it up and running, but again, the source code is your documentation for now.. You are of course welcome to ask for assistance here on the forum.

@mmontag: Is it OK to re-license your contributions under the LGPLv2.1+ license?

**Reply by bachimusprime** [reply]
Actually, the licensing of PCBox is just a little bit unclear. Certain parts (the parts I wrote) I know are under GPLv2+, but much of it is only known to be GPLv2, without necessarily being "or later", so I have to be careful.

**Reply by skjelten** [reply]
I did not have time to change the license now for the v0.3.0 release, but I do intend to change the license to LGPLv2.1+ for libemusc before the next release.

**Comment by mmontag** [comment]
@skjelten yep, fine with me

**Comment by skjelten** [comment]
The libEmuSC library have now been relicensed to LGPL2.1+
